IOS-5743 Aptos make basic declarations

skibinalexander opened the pull request:
Базовое описание блокчейна aptos

skibinalexander committed:
IOS-5743 Make basic declarations

Balashov152 commented on BlockchainSdk/ExternalLinkProvider/AptosExternalLinkProvider.swift:
```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+struct AptosExternalLinkProvider: ExternalLinkProvider {
+    var testnetFaucetURL: URL?
```
Это тоже наверно надо, если testnet есть
Можно ниже переместить раз рассчитываемую

ba1ashov commented:
В ветко все норм ? Это разве не в develop ?

skibinalexander commented:
> В ветко все норм ? Это разве не в develop ?

Ага, делаю в отдельную ветку фичевую, чтобы потом все отревьюенное слить

skibinalexander committed:
IOS-5743 Fix coding keys

skibinalexander committed:
IOS-5743 Merge branch 'aptos/IOS-5496-aptos-integration' into aptos/IOS-5743-aptos-wallet-manager

skibinalexander committed:
IOS-5743 Merge branch 'aptos/IOS-5496-aptos-integration' into aptos/IOS-5743-aptos-wallet-manager

skibinalexander committed:
IOS-5743 Merge branch 'aptos/IOS-5496-aptos-integration' into aptos/IOS-5743-aptos-wallet-manager

# Conflicts:
#	BlockchainSdk.xcodeproj/project.pbxproj
#	BlockchainSdk/Common/Address/AddressTypesConfig.swift
#	BlockchainSdk/Common/Blockchain+AllCases.swift
#	BlockchainSdk/Common/Blockchain.swift
#	BlockchainSdk/Common/Derivations/DerivationConfigV1.swift
#	BlockchainSdk/Common/Derivations/DerivationConfigV2.swift
#	BlockchainSdk/Common/Derivations/DerivationConfigV3.swift
#	BlockchainSdk/Common/Factories/AddressServiceFactory.swift
#	BlockchainSdk/Common/Factories/EstimationFeeAddressFactory.swift
#	BlockchainSdk/Common/Factories/ExternalLinkProviderFactory.swift

skibinalexander committed:
IOS-5743 Merge branch 'aptos/IOS-5496-aptos-integration' into aptos/IOS-5743-aptos-wallet-manager

skibinalexander committed:
IOS-5743 Merge branch 'aptos/IOS-5496-aptos-integration' into aptos/IOS-5743-aptos-wallet-manager

skibinalexander committed:
IOS-5743 Merge branch 'aptos/IOS-5496-aptos-integration' into aptos/IOS-5743-aptos-wallet-manager

skibinalexander committed:
IOS-5743 Fix review comments
